src/screens/Settings/components/LanguageScreen/components/LanguageItem/LanguageItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Image, Pressable, Text, View } from 'react-native';
+
+import { Icons } from '@UI/Icons';
+
+import { styles } from './styles';
+
+interface Props {
+  icon: keyof typeof Icons;
+  language: 'en' | 'ua' | undefined;
```

Comment:
use TS operator instead of undefined

+add an eslint rule for checking/fixing this issue in future